shared/src/test/scala/za/co/absa/hyperdrive/shared/utils/TestClassLoaderUtils.scala

```diff
@@ -35,8 +35,10 @@ class TestClassLoaderUtils extends FlatSpec {
   }
 
   // If this test fails, you might have to check TestClassLoaderUtilsOther.jar and createTestClassLoaderUtilsOtherJar.sh
-  it should "return the singleton class for a given trait and fully qualified class name from other jar" in {
+  ignore should "return the singleton class for a given trait and fully qualified class name from other jar" in {
```

Comment:
It works on my machine. Does it always fail on your machine or is it flaky? Please try to un-ignore the test and see if the Github action would fail on this test